bpf/lib/nodeport.h

```diff
@@ -1548,6 +1552,10 @@ static __always_inline int dsr_reply_icmp4(struct __ctx_buff *ctx,
 	if (ctx_store_bytes(ctx, off + sizeof(ip), &icmp,
 			    sizeof(icmp), 0) < 0)
 		goto drop_err;
+#if DSR_ENCAP_MODE == DSR_ENCAP_NONE
+	if (ctx_store_bytes(ctx, off + sizeof(ip) + sizeof(icmp), &inner_ip_hdr, sizeof(inner_ip_hdr), 0) < 0)
```

Comment:
The RFC is mentioning that the IP headers of the original packet + 64 bit of the original datagram's data.

Could add those extra data? They are expected by some equipment to resolve as an example NAT session

https://www.rfc-editor.org/rfc/rfc792.html

Reply:
Thanks. I updated the change now.